debug.c

```diff
@@ -80,7 +80,11 @@ int debug_set_destination(char *dest, int log_type) {
 
     if (!strncasecmp(dest, "file:///", 8)) {
         if (log_type != LOG_TYPE_FTICKS) {
+#ifdef __CYGWIN__
+            debug_filepath = stringcopy(dest + 8, 0);
+#else
             debug_filepath = stringcopy(dest + 7, 0);
+#endif
```

Comment:
If that works on all platforms, that's fine by me (it's *your* project, I'm just contributing ;-))... I'm looking at the syslog side of things too, because that stuff needs to go into the event log for Windows, and the calls are likely to be different.